homeassistant/components/climate/services.yaml

```diff
@@ -94,3 +94,27 @@ set_swing_mode:
     swing_mode:
     description: New value of swing mode
     example: 1
+
+ecobee_set_fan_min_on_time:
```

Comment:
They will only show if the service is registered, ie if the ecobee platform is set up. It should stay.